pyocd/debug/cache.py

```diff
@@ -120,10 +131,15 @@ def read_core_registers_raw(self, reg_list):
         # Read uncached registers from the target.
         read_list = list(reg_set.difference(cached_set))
         reading_cfbp = any(r for r in read_list if r in self.CFBP_REGS)
+        reading_xpsr = not reading_cfbp and any(r for r in read_list if r in self.XPSR_REGS)
```

Comment:
What is the `not reading_cfbp` term for?

Reply:
Me pointlessly micro-optimising to make it a sort of "else", saving another list search. Will drop.